src/app/admin/websiteDataTable/WebsiteDataTable.tsx

```diff
@@ -210,27 +220,61 @@ const WebsiteDataTable: React.FC = () => {
         <>
             {errorMessage && <ErrorSecondaryText>{errorMessage}</ErrorSecondaryText>}
             {rows && (
-                <DataGrid
-                    rows={rows}
-                    columns={websiteDataColumns}
-                    editMode="row"
-                    rowModesModel={rowModesModel}
-                    onRowModesModelChange={setRowModesModel}
-                    onRowEditStop={handleRowEditStop}
-                    processRowUpdate={processRowUpdate}
-                    slots={{
-                        loadingOverlay: LinearProgress,
-                    }}
-                    slotProps={{
-                        toolbar: { setRows, setRowModesModel, rows },
-                    }}
-                    loading={isLoading}
-                    getRowHeight={() => 150}
-                    apiRef={dataGridRef}
-                />
+                <DataGridStyling>
+                    <DataGrid
+                        rows={rows}
+                        columns={websiteDataColumns}
+                        editMode="row"
+                        rowModesModel={rowModesModel}
+                        onRowModesModelChange={setRowModesModel}
+                        onRowEditStop={handleRowEditStop}
+                        processRowUpdate={processRowUpdate}
+                        slots={{
+                            loadingOverlay: LinearProgress,
+                        }}
+                        slotProps={{
+                            toolbar: { setRows, setRowModesModel, rows },
+                        }}
+                        loading={isLoading}
+                        getRowHeight={() => 150}
+                        apiRef={dataGridRef}
+                        getRowClassName={(params) =>
+                            (params.indexRelativeToCurrentPage + 1) % 2 === 0 ? "even" : "odd"
+                        }
+                        hideFooter
+                    />
+                </DataGridStyling>
             )}
         </>
     );
 };
 
+const DataGridStyling = styled.div`
```

Comment:
Is this the same as the one in PackingSlotsTable? Can we commonise this?

Reply:
yep